tutorials/Tutorial of QA for FAST.ipynb

```diff
@@ -0,0 +1,140 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# This is the tutorial of FAST(FMRIB's Automated Segmentation Tool) quality analysis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "FAST (FMRIB's Automated Segmentation Tool) segments a 3D image of the brain into different tissue types (Grey Matter, White Matter, CSF, etc.), whilst also correcting for spatial intensity variations (also known as bias field or RF inhomogeneities). FAST is based on a hidden Markov random field model and an associated Expectation-Maximization algorithm. The whole process is fully automated and can also produce a bias field-corrected input image and a probabilistic and/or partial volume tissue segmentation. It is robust and reliable, compared to most finite mixture model-based methods, which are sensitive to noise.\n",
+    "\n",
+    "A quality analysis picture showing the segmentation of a skull-removed brain into white matter, gray matter, and cerebrospinal fluid. Segmentation uses the function of FAST in FSL. You can use the command line below to call the FAST function in FSL for splitting. The file type is nifiti image.\n",
+    "\n",
```

Comment:
Include a cell containing your qa code, the reader should be able to see exactly what is happening. Also explain exactly where the png file(s) will be saved and what they will be called